scripts/sct_download_data.py

```diff
@@ -12,23 +12,51 @@
 
 from msct_parser import Parser
 import sct_utils as sct
 
+# Dictionary containing list of URLs for data names. Mirror servers are listed in order of priority.
+DICT_URL = {
+    'sct_example_data': ['https://osf.io/kjcgs/?action=download',
+                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20180525_sct_example_data.zip'],
+    'sct_testing_data': ['https://osf.io/yutrj/?action=download',
+                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20191031_sct_testing_data.zip'],
+    'PAM50': ['https://osf.io/u79sr/download',
+              'https://www.neuro.polymtl.ca/_media/downloads/sct/20191029_PAM50.zip'],
+    'MNI-Poly-AMU': ['https://osf.io/sh6h4/?action=download',
+                     'https://www.neuro.polymtl.ca/_media/downloads/sct/20170310_MNI-Poly-AMU.zip'],
+    'gm_model': ['https://osf.io/ugscu/?action=download',
+                 'https://www.neuro.polymtl.ca/_media/downloads/sct/20160922_gm_model.zip'],
+    'optic_models': ['https://osf.io/g4fwn/?action=download',
+                     'https://www.neuro.polymtl.ca/_media/downloads/sct/20170413_optic_models.zip'],
+    'pmj_models': ['https://osf.io/4gufr/?action=download',
+                   'https://www.neuro.polymtl.ca/_media/downloads/sct/20170922_pmj_models.zip'],
+    'binaries_debian': ['https://osf.io/bt58d/?action=download',
+                        'https://www.neuro.polymtl.ca/_media/downloads/sct/20190930_sct_binaries_linux.tar.gz'],
+    'binaries_osx': ['https://osf.io/msjb5/?action=download',
+                     'https://www.neuro.polymtl.ca/_media/downloads/sct/20190930_sct_binaries_osx.tar.gz'],
+    'course_hawaii17': 'https://osf.io/6exht/?action=download',
+    'course_paris18': ['https://osf.io/9bmn5/?action=download',
+                       'https://www.neuro.polymtl.ca/_media/downloads/sct/20180612_sct_course-paris18.zip'],
+    'course_london19': ['https://osf.io/4q3u7/?action=download',
+                        'https://www.neuro.polymtl.ca/_media/downloads/sct/20190121_sct_course-london19.zip'],
+    'course_beijing19': ['https://osf.io/ef4xz/?action=download',
+                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20190802_sct_course-beijing19.zip'],
+    'deepseg_gm_models': ['https://osf.io/b9y4x/?action=download',
+                          'https://www.neuro.polymtl.ca/_media/downloads/sct/20180205_deepseg_gm_models.zip'],
+    'deepseg_sc_models': ['https://osf.io/avf97/?action=download',
+                          'https://www.neuro.polymtl.ca/_media/downloads/sct/20180610_deepseg_sc_models.zip'],
+    'deepseg_lesion_models': ['https://osf.io/eg7v9/?action=download',
+                              'https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip'],
+    'c2c3_disc_models': ['https://osf.io/t97ap/?action=download',
+                         'https://www.neuro.polymtl.ca/_media/downloads/sct/20190117_c2c3_disc_models.zip']
+    }
```

Comment:
DICT_URL is still getting deleted/added, so it's going to end up make two copies of it when I rebase my PRs. Can you put it back down below `get_parser`?

Changing the name to uppercase should be okay.

Reply:
i can, but in that case i don't know how to implement [this](https://github.com/neuropoly/spinalcordtoolbox/pull/2700#pullrequestreview-408886355)

Reply:
That's inside of `get_parser()` so it should be fine -- it doesn't actually run until `main()` calls it.

Reply:
I know, that list really should be more prominent in this file. But we can do it in a small PR next week.

Reply:
hum, i am still getting the error (in 9840ff76f611518c288e3db8c5fdd2dce779e636):
```bash
 sct_download_data -h

--
Spinal Cord Toolbox (git-jca/download-data-9840ff76f611518c288e3db8c5fdd2dce779e636)

Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 113, in <module>
    res = main()
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 97, in main
    parser = get_parser()
  File "/Users/julien/code/sct/scripts/sct_download_data.py", line 32, in get_parser
    example=sorted(dict_url))
NameError: name 'dict_url' is not defined
```

Reply:
Oh it was a local variable. I should have tried it myself, sorry.

Two fixes:

1. Very quickly: `parser = get_parser()` -> `parser = get_parser(dict_url)`
2. You could probably move ` def main(args=None): ` *after* `dict_url`; then there will be a small patch on the `dict_url` line but the rest of its contents should stay put.

We'll clean this up better once we've merged our work fully!

Reply:
👍 i did the first suggestion 8ad4aee81c0a03b42f87cd5a03fb0eade07bf51c